components/blocks/FilesToCards.vue

```diff
@@ -119,41 +129,51 @@
           :key="'help-card-' + i"
           class="mx-3 my-3 px-3"
           variant="light"
-          accent="link"
+          accent="warning"
           width="medium"
         >
           <template #header>
             <span v-if="item.group" class="radius-0 tag is-link has-text-light"
-              ><span class="icon"><i class="mdi mdi-account-multiple" /></span
               ><abbr :title="item.group | expandAcronym">
                 {{ item.group }}
               </abbr></span
             >
-            <h2 class="card-title">{{ item.title }}</h2>
+            <h2 class="title has-text-black pt-3">{{ item.title }}</h2>
             <div v-if="item.date">Updated: {{ item.date }}</div>
-            <span v-if="item.authors" class="small has-text-link"
-              >Authors: {{ item.authors.map((a) => a.name).join(', ') }}</span
-            >
+            <span v-if="item.authors" class="subtitle has-text-black"
+              ><i class="mdi mdi-account-multiple p-1 m-1"></i>
+              <span
+                v-for="(author, index) in item.authors"
+                :key="author.github_user"
+                ><a :href="'https://github.com/' + author.github_user">{{
+                  author.name
+                }}</a
+                ><span v-if="index + 1 < item.authors.length">, </span></span
+              >
```

Comment:
```suggestion
              <span
                v-for="(author, index) in item.authors"
                :key="author.name"
                ><a v-if="author.github_user :href="'https://github.com/' + author.github_user">{{
                  author.name
                }}</a
                >{{ !author.github_user && author.name }}<span v-if="index + 1 < item.authors.length">, </span></span
              >
```

Yup! You can combine, just not on the same element, so you have to do something like the above.